Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonDataGridViewIconColumn.cs

```diff
@@ -122,7 +122,7 @@ public override object Clone()
         [Category(@"Data")]
         [Description(@"Set of extra icons to appear with control.")]
         [DesignerSerializationVisibility(DesignerSerializationVisibility.Content)]
-        public List<IconSpec> IconSpecs { get; }
+        public List<IconSpec?> IconSpecs { get; }
```

Comment:
I do not think that the `IconSpec` should be null in the list, it adds complexity where it is not needed